### ISSUE TITLE
The allow, deny and revokeBtn elements need an upgrade. All currently inaccessible and unexpected behaviour in revokeBtn customization.

### ISSUE THREAD

**Issue by jaydenmuzzin** [osano/cookieconsent#653]
The allow, deny and revoke elements should be upgraded.

Firstly, they are inaccessible. To be accessible, users should have the ability to tab into & use the spacebar/enter key and get a response. The allow, deny and revoke elements currently do not allow this. 

Being anchor elements by default, the allow and deny elements are tabbable, but you can not press the spacebar or enter key to give a consent response. The revoke element is not tabbable and the consent bar is unable to be reopened with the spacebar or enter key after giving a consent response.

Fortunately, the default allow and deny consent elements can be overridden to enable the behaviour of pressing spacebar/enter to give their consent response:

Default - Doesn't allow spacebar/enter to make choice: 
`elements: {                                                                                                                                     
     allow: '<a aria-label="allow cookies" tabindex="0" class="cc-btn cc-allow"></a>',
     deny: '<a aria-label="deny cookies" tabindex="0" class="cc-btn cc-deny"></a>'
}`

Custom - Does allow spacebar/enter to make choice:
`elements: {
    allow: '<button class="cc-btn cc-allow">Allow</button>',                                                       
    deny: '<button class="cc-btn cc-deny">Deny</button>'
}`

While you can also customize `revokeBtn` to be tabbable and open the consent bar by pressing spacebar/enter, like so:
`revokeBtn:'<button class="cc-revoke ">Cookie Policy</button>'`

(i.e. the same as the default except a button element), you lose the styling and functionality provided by the classes `cc-top cc-animate` and the color override as they are not appended to the element, which I think is unexpected. I did end up just copying these classes from the inspector to the customized element, however I believe you shouldn't have to do this.

To summarize, I believe that:
- The allow and deny elements should accept pressing space and enter by default to make a consent choice;
- `revokeBtn` should be tabbable and accept pressing space and enter by default to open the consent bar;
- The classes that get appended to the default `revokeBtn` div element should also be appended to a customized `revokeBtn` element

**Comment by oajulianclementson** [comment]
It is possible to customise the revoke button without losing styles by adding the `{{classes}}` token:
`revokeBtn:'<button class="cc-revoke {{classes}}">Cookie Policy</button>'`

I could only work this out by examining the source code, because unfortunately the docs page at https://www.osano.com/cookieconsent/documentation/javascript-api/ is missing all its `{{content}}` tokens, presumably due to some error in character escaping.
